pkg/component/component.go

```diff
@@ -166,7 +166,7 @@ func GetCurrent(client *occlient.Client, applicationName string, projectName str
 
 // PushLocal push local code to the cluster and trigger build there.
 // asFile indicates if it is a binary component or not
```

Comment:
comments need to explain what `files` parameter is.

Reply:
added